files/fail_back.py

```diff
@@ -37,119 +34,100 @@ def run(self, conf_file, log_file, log_level):
                  "var_file: %s \n"
                  "vault: %s \n"
                  "ansible_play: %s \n"
-                 "report log: /tmp/%s \n",
+                 "report log file: /tmp/%s\n",
                  target_host,
                  source_map,
                  var_file,
                  vault,
                  ansible_play,
                  report)
 
-        cmd = []
-        cmd.append("ansible-playbook")
-        cmd.append(ansible_play)
-        cmd.append("-t")
-        cmd.append(dr_clean_tag)
-        cmd.append("-e")
-        cmd.append("@" + var_file)
-        cmd.append("-e")
-        cmd.append("@" + vault)
-        cmd.append("-e")
-        cmd.append(" dr_source_map=" + target_host)
-        cmd.append("--vault-password-file")
-        cmd.append("vault_secret.sh")
-        cmd.append("-vvv")
+        dr_clean_tag = "clean_engine"
+        extra_vars_cleanup = " dr_source_map=" + target_host
+        command_cleanup = [
+            "ansible-playbook", ansible_play,
+            "-t", dr_clean_tag,
+            "-e", "@" + var_file,
+            "-e", "@" + vault,
+            "-e", extra_vars_cleanup,
+            "--vault-password-file", "vault_secret.sh",
+            "-vvv"
+        ]
 
-        cmd_fb = []
-        cmd_fb.append("ansible-playbook")
-        cmd_fb.append(ansible_play)
-        cmd_fb.append("-t")
-        cmd_fb.append(dr_tag)
-        cmd_fb.append("-e")
-        cmd_fb.append("@" + var_file)
-        cmd_fb.append("-e")
-        cmd_fb.append("@" + vault)
-        cmd_fb.append("-e")
-        cmd_fb.append(" dr_target_host=" + target_host
-                      + " dr_source_map=" + source_map
-                      + " dr_report_file=" + report)
-        cmd_fb.append("--vault-password-file")
-        cmd_fb.append("vault_secret.sh")
-        cmd_fb.append("-vvv")
+        dr_failback_tag = "fail_back"
+        extra_vars_failback = " dr_target_host=" + target_host +\
+                              " dr_source_map=" + source_map +\
+                              " dr_report_file=" + report
+        command_failback = [
+            "ansible-playbook", ansible_play,
+            "-t", dr_failback_tag,
+            "-e", "@" + var_file,
+            "-e", "@" + vault,
+            "-e", extra_vars_failback,
+            "--vault-password-file", "vault_secret.sh",
+            "-vvv"
+        ]
 
-        # Setting vault password
-        vault_pass = input(
-            INPUT + PREFIX + "Please enter vault password (In case of "
-            "plain text please press ENTER): " + END)
+        # Setting vault password.
+        vault_pass = input(INPUT
```

Comment:
I think that the previous style was more readable.

Reply:
Publishing the updated patch now.